manual/cypher/cypher-docs/src/docs/dev/ql/functions/index.asciidoc

```diff
@@ -283,3 +283,24 @@ include::split.asciidoc[leveloffset=+1]
 include::reverse.asciidoc[leveloffset=+1]
 
 include::tostring.asciidoc[leveloffset=+1]
+
+ifndef::asciidoctor[:leveloffset: 1]
+
+[[query-functions-udf]]
+== User-defined functions ==
+User-defined functions are written in Java, deployed into the database and are called in the same way as any other Cypher function.
```

Comment:
I think I disagree this section is about `functions` I think I would prefer to keep procedures out of the discussion here. UDFs are compared with procedures in the dedicated section on UDFs.